multiple_generate.sh

```diff
@@ -7,19 +7,18 @@ function clean() {
 function cloner() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  git clone $1 ${REPONAME}_temp &
+  git clone $1 ${REPONAME} &
 }
 
-function repoCleaner() {
+function genSubIndex() {
   URL_SPLIT=(${1//// })
   REPONAME=(${URL_SPLIT[3]//./ })
-  cd ${REPONAME}_temp
-  mkdir ../$REPONAME
-  cp -a docs/. ../$REPONAME/.
-  cd ..
-  rm -rf ${REPONAME}_temp
+  cd $REPONAME/docs
+  python $BASE/modules/scripts/genindex.py ./
```

Comment:
@pri22296 In order to run the script in docs we have to change the dir to the doc. if we need readme I have to run the script in rootdir. I would like to get your suggestion on this